naresh/src/main/java/com/lftechnology/phpjava/ems/utlis/PasswordHashGenerator.java

```diff
@@ -0,0 +1,54 @@
+package com.lftechnology.phpjava.ems.utlis;
+
+
+import javax.crypto.SecretKey;
+import javax.crypto.SecretKeyFactory;
+import javax.crypto.spec.PBEKeySpec;
+import java.security.SecureRandom;
+import java.util.Base64;
+
+public class PasswordHashGenerator {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS1118)